tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/ClassSymbolTest.java

```diff
@@ -167,4 +168,22 @@ public void testDistinctClasses() {
         assertEquals(clazz.kind(), CLASS);
         assertTrue(clazz.qualifiers().contains(Qualifier.DISTINCT));
     }
+
+    @DataProvider(name = "hasDefaultTestProvider")
+    public Object[][] hasDefaultTest() {
+        return new Object[][]{
+                {66, 7, true},
+                {67, 7, true},
+                {68, 7, false},
+                {69, 7, false}
+        };
+    }
+
+    @Test(dataProvider = "hasDefaultTestProvider")
+    public void testHasDefaults(int line, int col, boolean hasDefault) {
+        Symbol symbol = model.symbol(srcFile,  LinePosition.from(line, col)).get();
+        ClassFieldSymbol fieldSymbol = (ClassFieldSymbol) symbol;
+        assertEquals(fieldSymbol.hasDefaultValue(), hasDefault);
+
```

Comment:
```suggestion
```

Reply:
Removed in 56b1bcd76d10af1ed0cd6ce23d365c5e9326506d